### DISCUSSION TITLE
Authelia and 2FA registration device

### DISCUSSION THREAD

**Discussion by PhilippeAMB** [authelia/authelia#6481] [Troubleshooting] [answered]
Hi

When i'm log on Authelia password prompt, it's ok
When i click on register device, email sent.

When i click on the link contained in the email, URL does not include port .

My website is https://authelia.mydomain.duckdns.org:1000/
the link contained in the email containt only https://authelia.mydomain.duckdns.org/one-time-password/register?xxxxxxxxx

Could you help me, i don't find.
thanks

my config : SWAG/F2ban/Authelia, all in docker

**Comment by james-d-elliott** [comment, marked as answer]
We expect X-Forwarded-Host header to include the entire host header i.e. hostname and port if relevant (per standard convention), not just the hostname portion. You'll need to correct that to resolve it. See the proxy.conf example: https://www.authelia.com/integration/proxies/nginx/#proxyconf

**Comment by PhilippeAMB** [comment]
Happy new year :)

Thanks very much, i made modification and it works.

**Comment by james-d-elliott** [comment]
Happy new year to you.
